frontend/src/content/news.json

```
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-04-04",
+      "title": "Metabolic Atlas 2.5 improves the about pages and fixes bugs",
+      "text": "<b>Metabolic Atlas 2.5</b> is out. In this minor release we have focused on maintenance, fixing a number of bugs related to search and routing. General improvements for the content pages, including the <i>About</i> pages and <i>Documentation</i>, have been added as well. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.5' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
+    },
     {
       "date": "2022-03-21",
       "title": "Metabolic Atlas 2.4 improves search functionality",
-      "text": "<b>Metabolic Atlas 2.4</b> is out. In this minor release we have focused on improving the search functionality. The improvement has been achieved for both the <i>GEM Search</i> and the <i>Global Search</i> by a better scoring of searching items and a more logical presentation of found results. Moreover, we have resolved a number of small bugs that affected the user experience. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.4' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
+      "text": "<b>Metabolic Atlas 2.4</b> is out. In this minor release we have focused on improving the search functionality. The improvement has been achieved for both the <i>GEM Search</i> and the <i>Global Search</i> by a better scoring of searching items and a more logical presentation of found results. Moreover, we have resolved a number of small bugs that affected the user experience. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.4' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/record/6378851#.Ykr4Fy8RpB0' target='_blank'>on Zenodo</a>."
```

Looking at the previous links, I don't think the anchor is needed.

```suggestion
      "text": "<b>Metabolic Atlas 2.4</b> is out. In this minor release we have focused on improving the search functionality. The improvement has been achieved for both the <i>GEM Search</i> and the <i>Global Search</i> by a better scoring of searching items and a more logical presentation of found results. Moreover, we have resolved a number of small bugs that affected the user experience. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.4' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/record/6378851' target='_blank'>on Zenodo</a>."
```

---

Ah that makes sense. Thanks! I was wondering why that part was constantly changing.